app/transport/rest/user.go

```diff
@@ -25,6 +28,28 @@ type User struct {
 	CreatedAt string `json:"created_at"`
 }
 
+// UsersResponse holds on array of Users are going to be rendered.
+type UsersResponse struct {
+	Results []User `json:"results"`
+}
+
+func cropUsers(solidUsers []entities.User) []User {
```

Comment:
Not obvious naming.
I would go with this option
```go
func toRESTUsers(users []entities.User) []User {
```
It's not the best naming as well, so you can try to find something better.